services/frontend-service/Earthfile

```diff
@@ -67,8 +68,11 @@ node-modules:
 
 lint-ui:
     FROM +deps-ui
-    RUN pnpm eslint-check
-    RUN pnpm scss-check
+    RUN pnpm eslint-fix
+    RUN pnpm scss-fix
+    SAVE ARTIFACT src/ui/components/ServiceLane/ServiceLane.scss AS LOCAL src/ui/components/ServiceLane/ServiceLane.scss
+    SAVE ARTIFACT src/ui/components/SideBar/SideBar.scss AS LOCAL src/ui/components/SideBar/SideBar.scss
+    SAVE ARTIFACT src/ui/components/dropdown/dropdown.scss AS LOCAL src/ui/components/dropdown/dropdown.scss
```

Comment:
this seems arbitrary. Why only these 3 files?

Reply:
rebase fumble, removed